src/components/feedback/index.tsx

```diff
@@ -1,4 +1,4 @@
-import PopUp from '#components/popUp'
+import PopUp from '#components/pop-up'
```

Comment:
Folders should be PascalCase: PopUp

filenames can be either PascalCase or camelCase
```suggestion
import PopUp from '#components/PopUp'
```

Reply:
this element is called a `Popper` from what I gather, with pop-ups reserved for new-window type objects

Reply:
Not worried about it.